chess/events/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@chess/events",
+  "version": "1.0.0",
+  "description": "Events for Chess game, to be imported into any frontend",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc -p tsconfig.json && tsc-alias -p tsconfig.json",
+    "lint": "prettier --write '**/*.ts'"
+  },
+  "keywords": [],
+  "author": "PaimaStudios",
+  "dependencies": {
+    "esrun": "^3.2.26",
+    "mqtt": "5.8.1",
```

Comment:
the client shouldn't need `mqtt` 
is this some workaround for something not compiling?

Reply:
same as ``esrun ?

Reply:
yeah, I forgot I did this. Since I'm mostly just copying the dependencies locally I added this to install it, but normally it would be a transitive dependency.